README.md

```diff
@@ -1,6 +1,6 @@
 # Shopgate Connect - Extension Product Image Badges
 
-Extension will allow a merchant to display a image badge on a product image based on a tag or property. If you wish to use a property then you must use extension [products-add-properties](https://github.com/shopgate/ext-products-add-properties) in order to have the property exposed in the same location as tags.
+Extension will allow a merchant to display an image badge on a product image based on a tag or property. If you wish to use a property then you must use extension [products-add-properties](https://github.com/shopgate/ext-products-add-properties) in order to have the property exposed in the same location as tags.
```

Comment:
could you please update the link to the extension? the readme of the the currently linked extension tells that it's deprecated and that instead "products-properties" is supposed to be used https://github.com/shopgate-professional-services/ext-products-properties